LiskHQ/lisk-hub
# Implement the new error style - closes #2543

### What issue have I solved?

-- #2543

### How have I implemented/fixed it?

Update the Feedback component and Input component to the see the feedback information to look same as design.

The change is all about css and not js components, then confirm and check all the places where this is use continue working as before.

### How has this been tested?


1. Run the app.
2. Check all the pages where input and feedback component works.
3. Trigger the error (feedback) message depending of the input type and see that the error message show as the designs.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
